EDDI/EDDI.cs

```diff
@@ -108,11 +108,11 @@ public static EDDI Instance
         // Information obtained from the player journal
         public Commander Cmdr { get; private set; } // Also includes information from the configuration and companion app service
         public string Environment { get; set; }
-        public StarSystem CurrentStarSystem { get; private set; } = new StarSystem();
```

Comment:
Did you mean to revert these initializations?

Reply:
These were only necessary so that we could subscribe an event handler (we couldn't subscribe to something that wasn't initialized). So for the moment at least we can revert them.